sale_order_line_discount_validation/tests/test_sale_order_line_discount_validation.py

```diff
@@ -0,0 +1,86 @@
+from odoo import fields
+from odoo.tests.common import TransactionCase
+
+
+class TestSaleOrderLineDiscountValidation(TransactionCase):
+    def setUp(self):
```

Comment:
@csil300 Thanks for this. Could you change to SavepointCase as it is faster and change setup to setUpClass ? Thanks